### ISSUE TITLE
Update from 0.37.7 to 0.38.0 break local css for Dash components

### ISSUE THREAD

**Issue by EBoisseauSierra** [plotly/dash-core-components#380]
**TL;DR:** `0.37.7 -> 0.38.0` breaks `dcc` css styling.
----
I have styled some `dcc` in a local `.css` file — e.g.:
```css
.Select-control {
  width: 100%;
  font-family: sans-serif;
  height: 50px;
  border-radius: 0;
  font-size: 100%;
}

.Select-placeholder,
.Select-value-label {
  line-height: 50px;
}

.DateRangePicker,
.DateRangePickerInput {
  display: block;
}

.DateInput {
  min-width: 110px;
  width: 45%;
  text-align: center;
}

.DateInput__display-text {
  font-family: sans-serif;
  font-weight: normal;
}

.DateRangePickerInput__arrow{
  width: 10%;
  min-width: 24px;
  text-align: center;
}

.Select-menu-outer{
  font-family: sans-serif;
}
```

The file is stored in my `asset` folder. It contains other style for other elements too.
When I upgraded from `0.37.2` to `0.38.0`, the css file is still loaded (other elements do get styled), however, `dcc` are not styled anymore:

`pip install -Iv dash-core-components==0.37.2`: **OK** (right dropdown size)
![screenshot from 2018-11-13 15-31-26](https://user-images.githubusercontent.com/37387755/48424288-4249b600-e75a-11e8-94e8-954a7ade5740.png)

`pip install -Iv dash-core-components==0.38.0`: **ERROR** (dropdowns are styled as default, despite css)
![screenshot from 2018-11-13 15-31-04](https://user-images.githubusercontent.com/37387755/48424258-3100a980-e75a-11e8-9e8e-0599a21ac523.png)

**Comment by chriddyp** [comment]
@valentijnnieman Do you know what's going on here? Did the CSS rules change in our upgrade or is our new CSS-in-JS method just placing our component CSS _after_ user-supplied CSS?

**Comment by valentijnnieman** [comment]
Looks like `style-loader` inserts the CSS at the bottom by default - causing user specified CSS (in `/assets`) to be loaded before, preventing it from overwriting the defaults. Thanks for reporting! I'll get a fix up for this as soon as possible.
